test/OpenTelemetry.Tests/Logs/LoggerProviderBuilderExtensionsTests.cs

```diff
@@ -134,8 +134,22 @@ public void LoggerProviderBuilderAddProcessorTest()
 
         using (var provider = Sdk.CreateLoggerProviderBuilder()
             .AddProcessor<CustomProcessor>()
-            .AddProcessor(sp => new CustomProcessor())
-            .AddProcessor(new CustomProcessor())
+            .AddProcessor(new CustomProcessor()
```

Comment:
I understand this is internal, but we want to make sure that - the current behavior is retained as-is. i.e processors are called in the same order as they were added.

Reply:
I don't think the tests as they were verified any order, only that the expected number of things were added. I updated to add more verification